modules/javafx.graphics/src/main/java/javafx/css/Selector.java

```diff
@@ -42,6 +42,12 @@
  */
 abstract public class Selector {
 
+    /**
+     * Creates a default {@code Selector}.
+     */
+    public Selector() {
+    }
+
     private static class UniversalSelector {
```

Comment:
Is a public constructor suitable in this class? `createSelector(String)` is the factory. There are public abstract methods here, on the other hand, so I don't know what the design idea is. The class docs should be updated too to say how to use this class.